frontend/frontend-react/src/components/EditRackForm.js

```diff
@@ -55,13 +56,13 @@ export class EditRackForm extends Component {
   render() {
     return (
       <div>
-        <Button variant="outlined" onClick={() => this.props.sendShowTable(true)}>Back</Button>{' '}
+        <Link to='/racks/'><Button variant="outlined">Back</Button>{' '}</Link>
         <form onSubmit={this.handleSubmit}>
-          <Grid container spacing={1}>
+          <Grid container spacing={2}>
             <Grid item xs={12}>
               <h1>Update Rack</h1>
             </Grid>
-            <Grid item xs={12}>
+            <Grid item xs={6}>
               <TextField shrink label='Updated Rack Number' type="text" fullWidth value={this.state.rack_number}
                          onChange={e => this.setState({rack_number: e.target.value})}/>
             </Grid>
```

Comment:
This doesn't work on my machine- am I doing something wrong?
https://www.loom.com/share/f80371ba7cae4dc1b0d2d29b75c3095f

Reply:
oops forgot to change that to a link instead of state heh